src/cvode/cvode_ls.c

```diff
@@ -1054,7 +1071,7 @@ int cvLsDQJac(realtype t, N_Vector y, N_Vector fy,
   } else if (SUNMatGetID(Jac) == SUNMATRIX_BAND) {
     retval = cvLsBandDQJac(t, y, fy, Jac, cv_mem, tmp1, tmp2);
```

Comment:
Do `cvLsBandDQJac` and `cvLsDenseDQJac` above not require error-checking round their calls?